inc/model.class.php

```diff
@@ -797,16 +803,14 @@ public function getTabNameForItem(CommonGLPI $item, $withtemplate = 0)
             switch ($item->getType()) {
                 case __CLASS__:
                     $tabs[1] = __('Model');
-                    if (!$this->isNewID($item->fields['id'])) {
+                    if (!$this->isNewID($item->fields['id']) ?? -1) { /** @phpstan-ignore-line */
```

Comment:
As there is only one case, I think the switch can become an if
This should simplify the code and make it possible to remove the phpstan-ignore

```
        if (!$withtemplate && $item->getType() == __CLASS__) {
            $tabs[1] = __('Model');
            if (!$this->isNewID($item->fields['id'] ?? -1)) {
```